File: config.example.js
// Reparado – Server-Anbindung
// -----------------------------------------------------------------------------
// So aktivierst du den echten, geteilten Betrieb (Reparado-Backend auf Hetzner):
//   1) Diese Datei nach  config.js  kopieren
//   2) apiUrl auf die Adresse deines Reparado-Backends setzen
//   3) Fertig – die App zeigt dann einen Login und arbeitet mit geteilten Daten.
// Ohne config.js läuft die App im lokalen Prototyp-Modus (localStorage).
//
// Hier stehen KEINE Geheimnisse – nur die öffentliche API-Adresse.
// Login/Rechte laufen über das Backend (Token). config.js ist in .gitignore.

window.REPARADO_CONFIG = {
  apiUrl: "https://werkstatt.margency.de"   // Adresse des Reparado-Backends (Caddy -> server.py)
  // Lokaler Test:  apiUrl: "http://127.0.0.1:8790"
};

File: config.example.py
# Reparado-Backend – Server-Konfiguration
# ---------------------------------------------------------------------------
# Auf dem Hetzner-Server nach  config.py  kopieren und ausfüllen.
# NICHT ins (öffentliche) Repo committen – steht in .gitignore.

# Sicherheit
SECRET_KEY = "HIER-LANGEN-ZUFALLSSTRING-EINSETZEN"     # signiert Login-Tokens (z. B. `openssl rand -hex 32`)
ADMIN_KEY  = "HIER-ZWEITEN-ZUFALLSSTRING-EINSETZEN"    # schützt /api/admin/provision (Werkstatt anlegen)

# Betrieb
DB_PATH     = "/opt/reparado/reparado.db"
PORT        = 8790
CORS_ORIGIN = "https://singh-oss.github.io"            # exakte Adresse der App (Frontend). "*" nur zum Testen.
TOKEN_TTL_DAYS = 30

# E-Mail-Versand (echter Kundennachrichten-Versand). Leer lassen = wird nur protokolliert.
SMTP_HOST = ""                 # z. B. "smtp.ionos.de"
SMTP_PORT = 587
SMTP_USER = ""                 # Postfach-Login
SMTP_PASS = ""                 # Postfach-Passwort
MAIL_FROM = "Dein Handy Doc <info@deinhandydoc.de>"

# Abrechnung / Stripe (Abo Basic/Pro/AI). Leer lassen = Abo deaktiviert (App läuft ohne Bezahlschranke).
APP_BASE_URL = "https://velqio.de"          # für Checkout-Rücksprung + Passwort-Reset-Links
STRIPE_SECRET_KEY    = ""                    # sk_live_… (GEHEIM! nur hier auf dem Server)
STRIPE_WEBHOOK_SECRET = ""                   # whsec_… (aus dem Webhook-Endpunkt im Stripe-Dashboard)
STRIPE_PRICE_BASIC = ""                      # price_… (Basic 49,90 €/Monat, wiederkehrend)
STRIPE_PRICE_PRO   = ""                      # price_… (Pro 84,90 €/Monat)
STRIPE_PRICE_AI    = ""                      # price_… (AI 99,90 €/Monat)
